### PR TITLE
Add more co-op copy

### PR DESCRIPTION
**Pull request by RobbieZhuang**
Added new copy up to "Does X affect job pay?" (that was enough work for 1 plane ride)

✨ This PR is best reviewed commit-by-commit ✨ 

Screen Shot 2023-04-22 at 10 04 11 PM
Screen Shot 2023-04-22 at 10 04 16 PM
Screen Shot 2023-04-22 at 10 04 19 PM
Screen Shot 2023-04-22 at 10 04 23 PM
Screen Shot 2023-04-22 at 10 04 26 PM
Screen Shot 2023-04-22 at 10 04 30 PM

### CODE REVIEW

File: data/coop.ts [hunk 1]
```diff
@@ -671,47 +671,47 @@ let INTERVIEWS_BOMBED = [
 
 const COOP_RATINGS = [
   {
-    group: "Co-op 1",
+    group: "1st Co-op",
```
Comment (wesley-a-leung) [inline data/coop.ts:674]: Any particular reason this group and the 6th group are called `1st Co-op` while groups 2 to 5 are just called `2nd`? Same thing with some of the other stats.

File: content/coop.html [hunk 2]
```diff
@@ -37,18 +36,19 @@ <h3>Companies</h3>
 <h3>Locations</h3>
 <p class="full">
   <span class="big">
-    Co-ops occured all across North America
+    Co-ops happend all across North America, and a few across the world!
```
Comment (wesley-a-leung) [inline content/coop.html]: happend -> happened

**Review by wesley-a-leung — APPROVED** [review]
Few minor comments

File: data/coop.ts (hunk 1 above)
Comment (RobbieZhuang) [reply data/coop.ts:674]: That's a great question! I thought it looked pretty good stylistically as the x-axis label for the graphs (screenshot below) - but also open to making it so only the first co-op has the `Co-op` tag or adding it to every label if that seems more clear/consistent (down to update that in a follow-up)!

Screen Shot 2023-04-22 at 9 33 13 PM

File: content/coop.html (hunk 2 above)
Comment (RobbieZhuang) [reply content/coop.html]: Ty for the reviewsss!!

File: data/coop.ts (hunk 1 above)
Comment (wesley-a-leung) [reply data/coop.ts:674]: I think this looks good as it is.

Comment (wesley-a-leung) [reply data/coop.ts:674]: I suppose another alternative is to have a secondary label underneath the x-axis labels that says "Co-op term" similar to some of the other charts.

**Pull request merged**